web/src/stream_list.ts

```diff
@@ -1044,33 +1047,18 @@ export function searching(): boolean {
     return $(".stream-list-filter").expectOne().is(":focus");
 }
 
-export function clear_search(e: JQuery.ClickEvent): void {
-    e.stopPropagation();
```

Comment:
Do you know what this `stopPropagation` was here? It's not obvious to me why it should or should not still be.

Reply:
`clear_search` was being called for a click, I moved the `stopPropagation` to `$("#clear_search_stream_button").on("click", (e: JQuery.ClickEvent) => {` below